cmake/addons/CMakeLists.txt

```diff
@@ -254,11 +254,16 @@ foreach(addon ${addons})
     # try to find a perfect match
     list(FIND ADDONS_TO_BUILD ${id} idx)
     if(idx GREATER -1 OR "${ADDONS_TO_BUILD}" STREQUAL "all")
+      message(STATUS "Exact Match ${id}, building addon")
```

Comment:
Yes, there the name not 100% correct if the `all` is used.

Reply:
As mentioned above, will just drop this change